deployment/app.ts

```diff
@@ -1,18 +1,27 @@
 #!/usr/bin/env node
 import "source-map-support/register";
-import * as cdk from "aws-cdk-lib";
+import { App, DefaultStackSynthesizer, Tags } from "aws-cdk-lib";
 import { EmptyParentStack } from "./stacks/empty/parent";
 import { ImportsIncludedParentStack } from "./stacks/imports_included/parent";
 import { ParentStack } from "./stacks/parent";
 import { determineDeploymentConfig } from "./deployment-config";
-import { getSecret } from "./utils/secrets-manager";
 import { getDeploymentConfigParameters } from "./utils/systems-manager";
 
 async function main() {
-  const app = new cdk.App({
-    defaultStackSynthesizer: new cdk.DefaultStackSynthesizer(
-      JSON.parse((await getSecret("cdkSynthesizerConfig"))!)
-    ),
+  const app = new App({
+    defaultStackSynthesizer: new DefaultStackSynthesizer({
+      deployRoleArn:
+        "arn:${AWS::Partition}:iam::${AWS::AccountId}:role/delegatedadmin/developer/cdk-${Qualifier}-deploy-role-${AWS::AccountId}-${AWS::Region}",
```

Comment:
What is Qualifier? Haven't seen that before.

Reply:
Im not really sure, in all of our accounts it seems to be hnb659fds.  I'm assuming AWS does or imagines making it something else sometimes.